### ISSUE TITLE
feature request: Setting to organize notifications and unread count in taskbar

### ISSUE THREAD

**Issue by nmat** [wavebox/waveboxapp#526]
* Which version of Wavebox are you using?
3.7.2beta

* Which Operating System are you using?
Windows 10

* Description:
This is a feature request to allow a better preference for unread count in taskbar and notifications.

The main reason is to allow notification for all services but in wavebox say what is going to be shown on desktop.

Situations where this can be good:
* Home (and ignore desktop notifications for services)
* Work (ignore personal account account notifications but force to show notifications fore work related items)

Another reason is that for example Teams or a indexed service sometimes do not show notifications. Telling what is allowed inside wavebox to send notifications would also allow a more advanced way of telling the machine what is allowed to run important notifications.

**Comment by Thomas101** [comment]
Yeah - sounds good. We've got Notification muting on our board, which will give some tooling towards this, but I think we can extend this further.
